scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -134,6 +148,8 @@ def main():
     )
     args = parser.parse_args()
 
+    if not GITHUB_TOKEN:
+        GITHUB_TOKEN = get_access_token_for_app()
```

Comment:
Missing arguments?

Reply:
Bah, clearly I didn't test this part of the change. Fixed.